ls1intum/Artemis
# `Assessment`: Show ∞ symbol for grading instructions without limit

### Checklist
#### General



#### Client

### Motivation and Context



### Description

Reducing confusion for displaying '0' as the value for unlimited applicable grading instructions by exchanging it with infinity symbol '∞'.

### Steps for Testing

Prerequisites:
- 1 Editor/Instructor
- 1 Programming/Modeling/Text/File-Upload Exercise with Structured Grading Criteria

1. Create an Exercise 
2. Create grading criteria with `Limit` greater than 0
3. Create grading criteria with `Limit` equal to 0 or undefined `Limit`
4. See that the value for undefined or equal to 0 `Limit` is now an infinity symbol '∞', otherwise the `Limit` is displayed as the defined number

### Review Progress





#### Code Review
#### Manual Tests

### Screenshots


Updated version:

<img width="279" alt="Bildschirmfoto 2021-12-14 um 12 55 21" src="https://user-images.githubusercontent.com/73841445/145993756-47aab91d-525d-4158-82b2-9dde4d16d9f0.png">

Old version:

<img width="392" alt="Components of structured assessment" src="https://user-images.githubusercontent.com/73841445/145994284-77d3ce9d-8553-4b96-9fb0-95833c67cef2.png">
